src/coreclr/jit/utils.cpp

```diff
@@ -3436,6 +3436,11 @@ uint32_t BitOperations::Log2(uint64_t value)
 // Return Value:
 //    The population count (number of bits set) of value
 //
+#if defined(_MSC_VER)
```

Comment:
note that this disables inlining of these methods too, so we might need to disable the specific optimization if this is perf critical.